.github/ISSUE_TEMPLATE/v8_breaking_change.md

```diff
@@ -11,15 +11,15 @@ assignees: ''
 
 **Which release will ship the breaking change?**
 
-<!-- e.g., v7.6.2 -->
+8.0
 
 **Describe the change. How will it manifest to users?**
 
 **What percentage of users will be affected?**
 
 <!-- e.g., Roughly 75% will need to make changes to x. -->
```

Comment:
This one is hard to answer if we don't have telemetry data, which is unfortunately rather common. I feel like we should be differentiating "guesses" from "data-driven" percentages... Thoughts? I answered the following when creating https://github.com/elastic/kibana/issues/81674

> Based on data: ??
> Based on hunches: a lot

Reply:
Good point. I think we should change this question to be "What's the impact of this breaking change?" and then use the comment to prompt the author to explain their reasoning and whether they have data to measure the impact or some other heuristic.

Reply:
Cool, sounds good. Lemme give it a shot real quick